frontend/src/pages/Guests/Guests.tsx

```diff
@@ -240,6 +215,22 @@ export const Guests = () => {
           />
         </Modal>
       ) : null}
+      {isEmailModalOpen ? (
+        <Modal
+          open={isEmailModalOpen}
+          onClose={() => {
+            setIsEmailModalOpen(false);
+          }}
+        >
+          <GuestInvite
+            initialValues={selectedEmail}
```

Comment:
I think if it's not going to be an object, we can maybe rename this prop from `initialValues` to maybe just `email` or `selectedEmail`